pkgs/development/python-modules/pylnk3/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, twine
+, invoke
+}:
+
+buildPythonPackage rec {
+  pname = "pylnk3";
+  version = "0.4.2";
+
```

No support for [Python < 3.6](https://github.com/strayge/pylnk/blob/master/setup.py#L32).

```suggestion

  disabled = pythonOlder "3.6";
```